components/omega/doc/design/VertCoord.md

```diff
@@ -0,0 +1,264 @@
+(omega-design-vert-coord)=
+# Vertical Coordinate
```

Comment:
One problem we've had in MPAS-Ocean that this document perpetuates is the arbitrary use of level vs. layer.  Do we want to settle on one of these, once an for all, and stick with it?  My (fairly strong) preference is for layer over level.

Reply:
I also prefer layer

Reply:
I agree we should settle on something consistent and I also would prefer layer.

Reply:
I agree as well.